react/features/base/media/reducer.js

```diff
@@ -136,5 +175,6 @@ function _video(state = VIDEO_INITIAL_MEDIA_STATE, action) {
  */
 ReducerRegistry.register('features/base/media', combineReducers({
     audio: _audio,
+    sounds: _sounds,
```

Comment:
as I said, I think this belongs in features/base/sounds, base/media is mostly concerned about what media types we use in a call.